src/lib/markbind/nunjuckUtils.js

```diff
@@ -0,0 +1,28 @@
+const START_ESCAPE_STR = '{% raw %}';
+const END_ESCAPE_STR = '{% endraw %}';
+const regex = new RegExp('({%( *)raw( *)%})(.*?)({%( *)endraw( *)%})', 'gs');
+
+function addEscapeTags(match) {
+  return START_ESCAPE_STR + match + END_ESCAPE_STR;
```

Comment:
I personally prefer string interpolation to explicit concatenation:
`${START_ESCAPE_STR}${match}${END_ESCAPE_STR}`;